### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: js/widgets.js [hunk 1]
```diff
@@ -123,7 +123,9 @@ function embedDatasets (territories, datasets) {
       }
     })
     const url = `${baseURL}/api/1/oembeds/?references=${references}`
-    // Polyfill added by webpack.
+    // Warning: if you are tempted to use generators instead of chaining
+    // promises, you'll have to use babel-polyfill which adds 300 Kb
+    // once the file is converted to ES5.
```
Comment (noirbizarre) [inline js/widgets.js]: ;)

File: udata/features/territories/commands.py [hunk 2]
```diff
@@ -71,8 +71,13 @@ def collect_references_files():
                     continue
                 filename = resource.url.split('/')[-1]
                 reference_path = references.path(filename)
+                log.info('Found reference: {reference_path}'.format(
+                    reference_path=reference_path))
```
Comment (noirbizarre) [inline udata/features/territories/commands.py]: Or you can use the logging extrapolation:
```python
log.info('Found reference: %s', reference_path)
```
Extrapolation is only evaluated on display

**Comment by noirbizarre** [comment]
r+wc

File: udata/features/territories/commands.py (hunk 2 above)
Comment (davidbgk) [inline udata/features/territories/commands.py]: Better 👍

**Pull request merged**